src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/equalsavoidnull/Example2.java

```diff
@@ -0,0 +1,24 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="EqualsAvoidNull">
+      <property name="ignoreEqualsIgnoreCase" value="true"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.equalsavoidnull;
+
+// xdoc section -- start
+public class Example2 {
+  public void foo() {
+    String nullString = null;
+    // violation below 'String literal expressions should be on the left side'
```

Comment:
Please add empty line above this line to make it easier to read in web

Reply:
done